docs/user/canonical-urls.rst

```diff
@@ -3,21 +3,34 @@ Canonical URLs
 
 A `canonical URL`_
 allows you to specify the preferred version of a web page to prevent duplicated content.
-They are mainly used by search engines to link users to the correct
-version and domain of your documentation.
+Here are some examples of when a canonical URL is used:
 
-If canonical URL's aren't used,
+- Search engines use your canonical URL to link users to the correct version and domain of your documentation.
+- Many popular chat clients and social media networks generate link previews,
+  using your canonical URL as the final destination.
+
+If canonical URLs aren't used,
 it's easy for outdated documentation to be the top search result for various pages in your documentation.
 This is not a perfect solution for this problem,
 but generally people finding outdated documentation is a big problem,
 and this is one of the suggested ways to solve it from search engines.
 
 .. _canonical URL: https://developers.google.com/search/docs/advanced/crawling/consolidate-duplicate-urls
 
+.. tip::
+
+   In most cases, Sphinx projects will automatically generate a canonical URL.
```

Comment:
```suggestion
   In most cases, Read the Docs will automatically generate a canonical URL for Sphinx projects.
```

Reply:
Less passive voice! :)

Reply:
Ah yeah, I get this part now - we pass the required http host information to Sphinx in order for it to be able to generate the Canonical URL :+1: